rosdep/python.yaml

```
@@ -5699,6 +5703,11 @@ python3-connexion-pip:
   ubuntu:
     pip:
       packages: [connexion]
+python3-construct:
+  arch: [python-construct]
+  debian: [python3-construct]
+  gentoo: [dev-python/construct]
```

Here is the Fedora package: https://src.fedoraproject.org/rpms/python-construct#bodhi_updates

```suggestion
  fedora: [python3-construct]
  gentoo: [dev-python/construct]
```

---

thanks :+1: